src/routes/auth.routes.js

```diff
@@ -1,14 +1,17 @@
 import { Router } from "express";
+import { authController } from '../di-container'
 import { getConfiguredPassport } from "../auth/passport"
-import googleAuth from "../controllers/auth.controller"
 
-let router = Router()
-let passport = getConfiguredPassport()
+const router = Router()
+const passport = getConfiguredPassport()
 
 router.get('/auth/google/callback', passport.authenticate("google", {
     scope: ["profile", "email"],
     session: false,
     failureRedirect: '/api/v1/auth/google/callback'
-}), googleAuth)
+}), authController.googleAuth)
+
+router.post('/logout', authController.logout)
```

Comment:
@ryanolee Actually, I was a bit unsure whether the route’s should go under GET or POST.  However, I decided on POST since the logout function might modify the database User table under some conditions (i.e. setting refresh token to null).  Is it still correct to use GET even with some potential db-data changes?

Reply:
The main reason to use POST is to send a HTTP request body to create a new DB entity. Here we are just preforming and operation that internally mutates the DB state. But considering we are not specifically creating a new entity (And not targeting one with a DELETE https://reqbin.com/Article/HttpDelete request)  The operation that probably lines up / is the easiest to use would be a GET method given it would make the endpoint directly accessible if we needed to use it in that manner in the future over relying on custom JS from the SPA to perform the logout operation.  🤔

Reply:
@ryanolee I see.  Thanks for th explanation.  Changed to GET in the upcoming PR update.